README.md

```diff
@@ -217,7 +217,6 @@ Argo uses Blacklight and ActiveFedora to expose the repository contents, and `do
 - RSolr
 - Sidekiq
 - Bootstrap
-- JQuery
```

Comment:
JQuery is no longer a key component of Argo. (Argo still depends on it, but only because Blacklight 7 does.)